addon/utils/determine-body-promise.ts

```diff
@@ -7,6 +9,10 @@ export default function determineBodyPromise(
   response: Response,
   requestData: JQueryAjaxSettings
 ): Promise<object | string | undefined> {
+  deprecate('determineBodyPromise is deprecated, it is no longer required for ember-data>=3.9.2', false, {
+    id: 'deprecate-fetch-ember-data-support',
+    until: '7.0.0'
+  });
```

Comment:
I dont think the utils should be deprecated because they're still useful for constructing one-off ajax requests using ember-fetch